bin/github-publish-release

```diff
@@ -1,67 +1,52 @@
 #!/usr/bin/env node
+import assert from 'assert';
+import { Octokit } from '@octokit/rest';
 
-const assert = require('assert');
-const GitHub = require('@octokit/rest');
-
-const repository = require('../lib/repository');
+import repository from '../lib/repository';
 
+// eslint-disable-next-line import/no-dynamic-require
 const pkg = require(`${process.cwd()}/package.json`);
 
-const checkErr = err => {
-  if (err) {
-    console.error(err.toJSON());
-    process.exit(1);
-  }
-};
-
-const github = new GitHub();
+const github = new Octokit();
 assert(process.env.GITHUB_OAUTH_TOKEN, 'GITHUB_OAUTH_TOKEN env variable should contain your personal access token');
-github.authenticate({ type: 'oauth', token: process.env.GITHUB_OAUTH_TOKEN });
+github.auth({ token: process.env.GITHUB_OAUTH_TOKEN, type: 'oauth' });
 
 const { user: owner, repo } = repository(pkg);
 
-github.repos.getLatestRelease({ owner, repo }, (err, { data: latestRelease }) => {
-  if (err) {
-    if (err.code === 404) {
-      latestRelease = { name: 'unknown', created_at: new Date('1970-01-01') };
-    } else {
-      checkErr(err);
-    }
+async function main() {
```

Comment:
is top-level async supported everywhere we're running this? (probably, just checking)

Reply:
what do you mean by "everywhere we're running this"? like in CI? yes it is :)